src/Components/EnvironmentPicker/EnvironmentPicker.tsx

```diff
@@ -188,7 +189,7 @@ const EnvironmentPicker = ({
         }
         if (isLocalStorageEnabled) {
             const selectedAdtInstanceInLocalStorage = getSelectedAdtInstanceFromLocalStorage(
-                localStorageKey
+                selectedItemLocalStorageKey
```

Comment:
why do we need this? And can we remove the localSotrageKey prop if we have this new key instead?

Reply:
These are all needed, but I noticed I was using the wrong key here.